lib/store/cleanup.go

```diff
@@ -29,19 +30,33 @@ import (
 
 // CleanupConfig defines configuration for periodically cleaning up idle files.
 type CleanupConfig struct {
-	Disabled bool          `yaml:"disabled"`
-	Interval time.Duration `yaml:"interval"` // How often cleanup runs.
-	TTI      time.Duration `yaml:"tti"`      // Time to idle based on last access time.
-	TTL      time.Duration `yaml:"ttl"`      // Time to live regardless of access. If 0, disables TTL.
+	Disabled            bool          `yaml:"disabled"`
+	Interval            time.Duration `yaml:"interval"`             // How often cleanup runs.
+	TTI                 time.Duration `yaml:"tti"`                  // Time to idle based on last access time.
+	TTL                 time.Duration `yaml:"ttl"`                  // Time to live regardless of access. If 0, disables TTL.
+	AggressiveThreshold int           `yaml:"aggressive_threshold"` // The disk util threshold to trigger aggressive cleanup. If 0, disables aggressive cleanup.
+	AggressiveTTL       time.Duration `yaml:"aggressive_ttL"`       // Time to live regardless of access if aggressive cleanup is triggered.
 }
 
+type (
+	// Define a func type for mocking diskSpaceUtil function.
+	diskSpaceUtilFunc func() (int, error)
+)
+
 func (c CleanupConfig) applyDefaults() CleanupConfig {
 	if c.Interval == 0 {
 		c.Interval = 30 * time.Minute
 	}
 	if c.TTI == 0 {
 		c.TTI = 6 * time.Hour
 	}
+
+	if c.AggressiveThreshold != 0 {
+		if c.AggressiveTTL == 0 {
+			c.AggressiveTTL = 30 * time.Minute
```

Comment:
In reality, 1 or 2 hours might be more reasonable defaultl

Reply:
Change to 1 hour.